<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">fontsize:30px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">	background-color: white;
	color: #000000;
	border-color: #000000;</string>
   </property>
   <widget class="QLabel" name="Calculo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange;
	color: #000;
	border-style: solid;
	border-radius: 0px;	
	border-color: #000000;
	font-size: 40px;
	padding: 10px;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suma">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="producto">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resta">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0f1a3b;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: #2346b0;
	color:white;

}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="igual">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="potenciar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>x^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="raizr">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrar_termino">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #7a0a1e ;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color:#d11739;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="division">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #7a0a1e ;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color:#d11739;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrar_undigito">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #7a0a1e ;
	color: white;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color:#d11739;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negativo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #cfcfcf;
	color: #2b2b2b;
	border-style: solid;
	border-radius: 15px;
	border-color: #000000;
	font-weight: bold;
	font-size: 18px;
	margin: 2px;
}

QPushButton::hover
{
	background-color: black;
	color:white;

}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
